Add test plans for lecturer interface (#114)
Gamify-IT/docs: dev-manuals/test-plans/lecturer-interface-frontend/plans/u-lecturer-interface-10-toggle-active-status-of-area.md

# Lecturer-Interface: Change active status of area (`u.lecturer-interface-10`)

Version: V1.0, 09.08.2022 \
Author: Max Kästner

## Description

The active status of a world or dungeon (which is linked to the world of a view) can be changed on the world view.

## Precondition

> At least one course is created. The tester is on the view of a world of a course (`/courses/<course-id>/worlds/<world-index>`)

Reviewer: ```suggestion
At least one course with a world is created. The tester is located where the area can be configured.
```

## Postcondition

The active status of an area was updated and is shown in the view.

## Typical procedure

1. Click the active switch button of an area (select world/dungeon by static name)

> 2. The switch button switched from ative to not active or from not active to active.

Reviewer: ```suggestion
2. The switch button inverted the `active` state.
```

> 3. Depends on the status before:
>     - In case the area was not active before, a green toast message appears which contains the area was activated
>     - In case the area was active before, a red toast message appears which contains the area was deactivated

Reviewer: ```suggestion
3. Depending on the previous state:
    - If the area was previously not active, a success message appears
    - If the area was previously active, a warning message appears
```
Also: Why not yellow?

Author: check answer above
